lobby.go

```diff
@@ -207,6 +207,50 @@ func wsListen(lobby *Lobby, player *Player, socket *websocket.Conn) {
 						lobby.Drawer.WriteAsJSON(JSEvent{Type: "your-turn"})
 					}
 				}
+			} else if received.Type == "on-kick-button" {
+				toKickID, isString := (received.Data).(string)
+				toKick := -1
+				if !isString {
+					fmt.Println("Invalid data")
+					continue
+				}
+
+				for index, otherPlayer := range lobby.Players {
+					if otherPlayer.ID == toKickID {
+						toKick = index
```

Comment:
add a break condition here for better performance. There is no need to keep iterating if we have already found the player.